main.js

```diff
@@ -449,7 +450,26 @@
                     fetchWPRDCData(dataSourceName);
                 });
                 retryDiv.appendChild(retryButton);
-            }));
+            }))
+            .then(() => {
+                return fetch(WPRDC_META_URL + dataSource.id)
+                .then((response) => {
+                    //check response for data and data for date
+                    if (response.status >= 200 && response.status < 300) {
+                        return response.json();
+                    }
+                })
+                .then((metadata) => {
+                    // Parse json for last modified field
+                    var parsedDate = Date.parse(metadata.result.last_modified);
+
+                    // Display a notification if the dataset has been updated within the hour
+                    var diff = Date.now() - parsedDate;
+                    if (diff <= 3600000) {
+                        displayNotification("The " + dataSourceName + " dataset has been recently updated.");
```

Comment:
This is fine, but for what it's worth the string templating you had in your previous commit was so close--in order to use the `string ${variable}` syntax, you need to use ` (backtick), rather than " or '